components/camel-cxf/camel-cxf-soap/src/main/docs/cxf-component.adoc

```diff
@@ -567,7 +567,13 @@ and setDefaultBus properties from spring configuration file.
 
 == Examples
 
-=== How to create a simple CXF service with POJO data format
+=== How to consume a message from a Camel CXF endpoint in POJO data format
+
+The Camel CXF endpoint consumer POJO data format is based on the
+http://cxf.apache.org/docs/invokers.html[CXF invoker], so the
+message header has a property with the name of
+`CxfConstants.OPERATION_NAME` and the message body is a list of the SEI
+method parameters.
```

Comment:
```suggestion
operation parameters.
```

SEI is not necessarily wrong, but requires some knowledge that is not that important here. "operation parameters" is perhaps easier to understand.

Reply:
Done